app/src/main/res/layout/password_fragment.xml

```diff
@@ -41,18 +42,20 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintHorizontal_bias="0.055"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/tvEnter" />
+        app:layout_constraintTop_toBottomOf="@+id/tvEnter"
+        android:layout_marginStart="@dimen/size20" />
 
     <EditText
         android:id="@+id/editPassword"
-        android:layout_width="match_parent"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_margin="@dimen/size30"
         android:ems="10"
-        android:hint="Enter Password"
+        android:hint="@string/password"
         android:textColorHint="@color/black"
         android:textColor="@color/black"
-        android:inputType="number"
+        android:inputType="numberPassword"
```

Comment:
@raghav32 we should not restrict user to have only number as their password. User can have text in their password as well